facebookresearch/ParlAI
# Change image_mode from none to no_image_model

**Patch description**
As in #1946, changes constant in teachers for image related tasks that use image_mode == 'none' to 'no_image_model'.

**Testing steps**
Ran on taks: flickr30k, mnist_qa, hopefully some vqa task later
Task: flickr30k
`python3 examples/display_data.py -t flickr30k` after changes, expected to see conversations. Did see conversations.
`python3 examples/train_model.py -t flickr30k -m memnn -mf /tmp/test_flickr30k --candidates fixed -vme 20 --image_mode no_image_model -vtim 200`
after changes, and same command but with `--image_mode none`. Expected outputs between both to be the same, got same output. 
Tried `--image_mode none` after changes, expected `ModuleNotFoundError: No module named 'torchvision'` (as that's what happens when you input models that don't exist)
 error and got the error.

Task: mnist_qa
 `python3 examples/train_model.py -t mnist_qa -m memnn -mf /tmp/test_mnistqa --candidates fixed -vme 20 --image_mode no_image_model -vtim 200 -dt train`
 Testing expectation similar as above.


**Logs**
(I can add some lines from the start of training / log of the error when there's an invalid image model later)

**Other information**
N/A

**Data tests (if applicable)**
`python tests/datatests/test_new_tasks.py`
Got output:
```
There are 591753 missing_text in coco_caption:DefaultTeacher.
There are 82783 missing_text in coco_caption:V2014Teacher.
There are 591753 missing_text in coco_caption:V2017Teacher.
There are 29000 missing_text in flickr30k:DefaultTeacher.
```
and stopped execution early (it had been running for a few hours).
